src/Saunter/AsyncApiMiddleware.cs

```diff
@@ -39,10 +39,10 @@ public async Task Invoke(HttpContext context)
 
             var asyncApiSchema = _asyncApiDocumentProvider.GetDocument(_options, prototype);
 
-            await RespondWithAsyncApiSchemaJson(context.Response, asyncApiSchema, _asyncApiDocumentSerializer, _options);
+            await RespondWithAsyncApiSchemaJson(context.Response, asyncApiSchema, _asyncApiDocumentSerializer);
         }
 
-        private static async Task RespondWithAsyncApiSchemaJson(HttpResponse response, AsyncApiSchema.v2.AsyncApiDocument asyncApiSchema, IAsyncApiDocumentSerializer asyncApiDocumentSerializer, AsyncApiOptions options)
```

Comment:
Was this change supposed to be here?

Reply:
> Was this change supposed to be here?

yes, not used parameter, her inject from DI

Reply:
Ooh it's private, didn't notice yesterday.